src/components/about/WhatMakesUsUnique.js

```diff
@@ -117,19 +118,18 @@ const WhatMakesUsUnique = () => {
       </Typography>
       <div className={cardsContainer}>
         {cardList.map(({ title, image, subTitle, textOnOver }) => (
-          <Grid key={title} container alignItems="center" className={cardContainer}>
+          <Grid key={title} container alignItems="center" className={`${muiGridWithOnHoverText} ${cardContainer}`}>
```

Comment:
Do we have `class-names` loaded as a module on the website? If not, this is fine.

Reply:
No, we don't yet
I am thinking about adding it (in a future PR)